Fix user feedback icon ui states

flozia opened the pull request:
## Description

Fixes the icon ui states in the “Give feedback” form.

https://user-images.githubusercontent.com/13835474/198679646-85613c5f-041a-4897-8b9c-53ee8446c691.mov

## Reference

- #4097

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [x] I have commented my code PARTICULARLY in hard to understand areas
- [x] I have added thorough tests where needed

MattLichtenstein commented:
Everything looks good, just one question

In #4097, the steps to reproduce and expected results say: 

**Step to repro**: Hover over each rating pictogram and observe the behavior;
**Expected result**: Each rating pictogram changes its color from gray to blue when hovering over. 

which appears to be the opposite of what is implemented. Implementation has a gray icon with blue outline on hover. Just trying to figure out if the expected results or the implementation is correct

flozia commented:
> Everything looks good, just one thought
> 
> In #4097, the steps to reproduce and expected results say:
> 
> **Step to repro**: Hover over each rating pictogram and observe the behavior; **Expected result**: Each rating pictogram changes its color from gray to blue when hovering over.
> 
> which appears to be the opposite of what is implemented. Implementation has a gray icon with blue outline on hover. Just trying to figure out if the expected results or the implementation is correct

Good point. Updated the icon to be blue on hover and only have an outline when selected and added a screencast to the description. Also checking with you @peterbenvenuto just to make sure this is the correct behaviour.
